### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 1]
```diff
@@ -1671,7 +1671,7 @@ rendering.macroCategory.Internal=Internal
 ### Macro Descriptors
 rendering.macro.groovy.name=Groovy
 rendering.macro.groovy.description=Execute a groovy script.
-rendering.macro.groovy.content.description=the groovy script to execute
+rendering.macro.groovy.content.description=The wiki content to execute.
```
Comment (surli) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Is this expected?

Comment (surli) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: @mflorea do you confirm that's wrong?

Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: @surli the issue is that the ``rendering.macro.groovy.content.description`` key is duplicated in oldcore's ``ApplicationResources.properties``. This commit seems to be the culprit https://github.com/xwiki/xwiki-platform/commit/7fee6198054656f2ec3d47246964e5f890ac4f30 . Bad copy & paste from @tmortagne ?

Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: So yes, it's wrong. We need to remove the duplicated key from ``ApplicationResources.properties``.

Comment (surli) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Aaaah thanks! Actually weblate was telling me an alert with: 
> Duplicated string for translation.

and many translations and I wasn't understanding the reason... now it's clear.

**Pull request merged**